subiquity/server/server.py

```diff
@@ -529,13 +532,13 @@ async def wait_for_cloudinit(self):
             self.cloud_init_ok = True
         log.debug("waited %ss for cloud-init", time.time() - ci_start)
         if "status: done" in status_txt:
-            log.debug("loading cloud config")
-            init = stages.Init()
-            init.read_cfg()
-            init.fetch(existing="trust")
-            self.cloud = init.cloudify()
```

Comment:
Other areas of the code reference self.cloud.  Probably best to leave these first few lines for now until we can revisit that.

Reply:
@dbungert Thanks for the review. I think I'll close this PR and put it on back burner until cloud-init also publishes full user-data/vendor-data content into instance-data-sensitive.json: merged_cfg key. Then it'll make this changeset worth it.

Reply:
Thanks @blackboxsw - I like where this changeset is going!